playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -224,6 +224,9 @@ EDXAPP_ENABLE_SYSADMIN_DASHBOARD: false
 EDXAPP_ENABLE_COURSE_DISCOVERY: true
 EDXAPP_ENABLE_DASHBOARD_TABS: true
 
+# Making Honor mode eligible for certificate
+ENABLE_CERTIFICATES_FOR_HONOR_MODE: true
```

Comment:
Should follow convention EDXAPP_ENABLE_CERTIFICATES_FOR_HONOR_MODE which is subsequently assigned to ENABLE_CERTIFICATES_FOR_HONOR_MODE

Reply:
updated.